Resources/Textures/Mobs/Aliens/Guardians/guardians.rsi/meta.json

```diff
@@ -0,0 +1,162 @@
+{
+  "version": 1,
+  "copyright": "taken from /tg/ station at commit https://github.com/tgstation/tgstation/commit/02756c2bc2cf3000080d030955e994242bab39b5",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "magic_flare",
+      "directions": 4,
+      "delays": [
+        [
+          1
+        ],
+        [
+          1
+        ],
+        [
+          1
+        ],
+        [
+          1
+        ]
+      ]
```

Comment:
Default delays not needed

Reply:
For the future if they are ever to be animated, which they likely will be

Reply:
I want to leave them in case they'll be animated, so you won't have to copy it all over again and fuck up the meta